hieunc-guestbook-dojo/static/js/guestbook/widget/_WidgetGuestBookEdit.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Created by hieunc on 25/11/2015.
+ */
+
+define([
+	"dojo/_base/declare",
+	"my/_ViewBaseMixin",
+	"dojo/text!./templates/_WidgetGuestBookEdit.html",
+	"dojo/cookie"
+], function (declare, _ViewBaseMixin, template, cookie) {
+	return declare("_WidgetGuestBookEdit", [_ViewBaseMixin], {
+
+		constructor: function (kwArgs) {
+			this.inherited("constructor", arguments);
```

Comment:
- sửa lại là `this.inherited(arguments);`
  - https://dojotoolkit.org/reference-guide/1.10/dijit/_WidgetBase.html#this-inherited